src/components/App/App.js

```diff
@@ -24,21 +26,45 @@ class App extends Component {
     .finally(() => this.setState({ isLoading: false }))
   }
 
+
+  searchByTitle = (searchCriteria) => {
+
+   let movieList
+
+    if(!isNaN(searchCriteria)) {
+      movieList = this.state.movies.filter(movie => {
```

Comment:
We'll need to fix this error in terminal for line 35: _Array.prototype.filter() expects a value to be returned at the end of arrow function_